src/components/TextInput/BaseTextInput.js

```diff
@@ -327,8 +332,7 @@ function BaseTextInput(props) {
 
                                     // Explicitly remove `lineHeight` from single line inputs so that long text doesn't disappear
                                     // once it exceeds the input space (See https://github.com/Expensify/App/issues/13802)
-                                    !isMultiline && {height, lineHeight: undefined},
-
+                                    !isMultiline && { height, lineHeight: Browser.isSafari() ? lineHeight : Browser.isMobileChrome() ? height : undefined},
```

Comment:
Why do we need to add changes for Chrome? There was no issue on chrome.

Reply:
In mobile chrome, line height issue exists as well. I have double checked and pasted video as well showing solution working as it is in mobile chrome as well. Please check on your end as well.

Reply:
@ayazalavi Can you please post a video showing issue on mobile chrome?

Reply:
Mobile chrome video is attached already. You have to test it on your PC to verify that it is working fine on all platforms.

Reply:
@ayazalavi I am requesting for the video showing the problem. I want to know what problem are we trying to fix on chrome.

Reply:
sure sharing that shortly.

Reply:
Please take a look at this video. In it, you can observe that the line height needs to be the same as the input height in order to prevent fluctuations. This behaviour is specific to mobile Chrome. In Safari, the Shadow DOM is calculated differently, and setting the maximum possible content height as our line height appears to resolve the issue.

https://github.com/Expensify/App/assets/767439/a0d2e0ff-8ba5-4050-accb-aaded8120408

Reply:
I see what the problem is. So, moving towards the solution, I am not sure why do we choose lineHeight for Safari and height for android? Is there any reason behind it? Can we use the same for both?

Reply:
Props.lineheight is getting passed so in Safari we can use it since it is enough lineheight. But in chrome we have larger contents so we need to use lineheight same as the input height. Hope that is clear enough. 

No we can't use same since problems happening are different in  mobile chrome and in safari.

Reply:
Sorry @ayazalavi . I don't get you. Can you please elaborate in detail?

Reply:
[BaseTextInput.js](https://github.com/Expensify/App/pull/27480/files#diff-511655d817607a5a2b12918a659da08129dbf4e74dc693f9a93f8ac9df7eb5ef) is receiving lineHeight as a prop. In most cases, we can use this value, but in mobile Chrome, the size of the contents varies significantly compared to Safari. Therefore, we need to use height as the line height in this case, which will prevent fluctuations.

Reply:
@sobitneupane I just added useMemo that is making code looks slightly more cleaner. Take a look at the new changes please.